src/App.js

```diff
@@ -1,13 +1,26 @@
 import React, { Component } from 'react';
-import PostCommentsDisplay from './PostCommentsDisplay/components/PostCommentsDisplay';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
+import Home from './Home/components/Home';
+import ListingDisplay from './ListingDisplay/components/ListingDisplay';
+import Navigation from './Navigation';
+
 import './App.css';
 
 class App extends Component {
```

Comment:
Consider making App a function. Since you never need to change the state, it can work as a function and would make for less code and easier test.

Reply:
Done